pkg/executor/kubernetes.go

```diff
@@ -357,7 +357,7 @@ func (k8s *k8s) Execute(command string) (TaskHandle, error) {
 	}
 
 	// Best effort potential way to check if binary is started properly.
-	taskHandle.Wait(100 * time.Millisecond)
+	taskHandle.Wait(10 * time.Second)
```

Comment:
@mpatelcz @skonefal @ppalucki - this is eeeeeeeeeeeeeevil but I had to increase wait time that much to make sure that Kubernetes executor returns error when non-existing command is executed.